src/app/tutorial.cljs

```diff
@@ -1,7 +1,7 @@
 (ns app.tutorial
   (:require
    [clojure.string :as string]
-   [sci.impl.vars :refer [SciVar]]))
+   [sci.impl.vars :refer [IVar]]))
```

Comment:
In the new version of sci `SciVar` has been removed, `IVar` it's something else.
I found that here we can require instead `[sci.lang :refer [Var]]` and use it for testing afterwards.

Reply:
@eliascotto Thank you for the correction - updated.